cli/src/cli.mjs

```diff
@@ -470,18 +476,49 @@ if ( parsed.help ) {
     });    
   }
 
-}
-  else if (parsed.getStatus){
-    axios.get(url(host, port, '/api/v1/status'), undefined)
-      .then(function (response) {
-        console.log(response.data);
-      })
-      .catch(function (error) {
-        logError(error);
+} else if (parsed.getStatus) {
+  axios.get(url(host, port, '/api/v1/status'), undefined)
+    .then(function (response) {
+      console.log(response.data);
+    })
+    .catch(function (error) {
+      logError(error);
     });
-  }
-  else {
-
+} else if (parsed.downloadOverrides) {
+  /* overrideDirectory is mock-firebolt/cli/externalOverrides by default, 
+  if overrided via CLI, will have user given location to save repository contents */
+  const overrideDirectory = parsed.overrideLocation ? parsed.overrideLocation : '../externalOverrides'
```

Comment:
Will this still save to the same location even if someone calls cli.mjs from a different location?

Ex: "node src/cli.mjs" vs "node cli.mjs"